### DISCUSSION TITLE
Public reviews

### DISCUSSION THREAD

**Discussion by gfellerph** [swisspost/design-system#1799] [General]
To make our work more transparent and create a possibility to interact with our stakeholders, I'd like to start doing public reviews. How does that work?

- Regular interval (monthly?)
- For all stakeholders (management, architecture, UI devs, UX designers, external employees, people with nothing else to do, ...), needs to be accessible without Post account (is Teams able to do this?)
- Predefined agenda, published some days before the review. Stakeholders can request topics and maintainers can set topics they'd like to show. 
- Time slot for questions and answers
- Not tied to the release process (releases should happen whenever they are ready)
- Moderated by a maintainer
- E.T. has public review pages on sharepoint with agenda topics and key figures https://postchag.sharepoint.com/sites/EKP-Next-Level2/SitePages/Public-Review---Sprint-74.aspx

Any other ideas?

**Comment by imagoiq** [comment]
Some thoughts:
- How long should the meeting be? 
- Should we record the meeting?
- An attractive title would be nice

**Reply by gfellerph** [reply]
1. I was thinking about an hour tops, maybe even just 30min for starting out
2. Probably a good idea if everyone is ok with it
3. Any suggestions?

**Comment by josemanosalvas** [comment]
About the schedule: the PostPortal team meets every Thursday onsite for the SCRUM ceremonies, so for us it would be easier to join if these public reviews are held any other day.

**Comment by gfellerph** [comment]
Open Office Hours established.
